docs/homepage/blog/offline_reinforcement_learning_algorithm_phase1/index.md

```diff
@@ -0,0 +1,315 @@
+- Time planning: the following is a relatively simple time table.
+  
+| Date       | Work    |
+| :-:       | :-:   |
+| Prior - June 30   | Preliminary research, including algorithm papers, ReinforcementLearning.jl library code, etc. |
+| The first phase |      |
+| July1 - July15  | Design and build the framework of offline RL.  |
+| July16 - July31 | Implement and experiment offline DQN and  offline SAC as benchmark. |
+| August1 - August15 | Write build-in documentation and technical report. Implement and experiment CRR.|
+| The second phase |      |
+| August16 - August31 | Implement and experiment PLAS. |
+| September1 - September15  | Research, implement and experiment new SOTA offline RL algorithms. |
+| September16 - September30 | Write build-in documentation and technical report. Buffer for unexpected delay. |
+| After project | Carry on fixing issues and maintain implemented algorithms.   |
+
+## Project Schedule
+This part mainly introduces the results of the first phase.
+
+#### Basic framework
+To run and test the offline algorithm, we first implemented `OfflinePolicy`([link](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningZoo/src/algorithms/offline_rl/common.jl)).
+```julia
+Base.@kwdef struct OfflinePolicy{L,T} <: AbstractPolicy
+    learner::L
+    dataset::T
+    continuous::Bool
+    batch_size::Int
+end
+```
+This implementation of `OfflinePolicy` refers to `QBasePolicy` ([link](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/src/ReinforcementLearningCore/src/policies/q_based_policies/q_based_policy.jl)). It provides a parameter `continuous` to support different action space types, including continuous and discrete. `learner` is a specific algorithm for learning and providing policy. `dataset` and `batch_size` are used to sample data for learning.
+
+Besides, we implement corresponding functions `π`, `update!` and `sample`. `π` is used to select the action, whose form is determined by the type of action space. `update!` can be used in two stages. In `PreExperiment` stage, we can call this function for pre-training algorithms with `pretrain_step` parameters (such as PLAS). In `PreAct` stage, we call this function for training the `learner`. In function `update!`, we need to call function `sample` to sample a batch of data from the dataset. With the development of RLDataset.jl, the `sample` function will be deprecated.
+
+We can quickly call the offline version of the existing algorithms with almost no additional code with this framework. Therefore, the implementation and performance testing of offline DQN and offline SAC can be completed soon. For example:
```

Comment:
I think this is the first place to mention **SAC**, better to add a reference here.